src/app/src/main/java/br/com/italoromeiro/arctouchtest/utils/PermissionUtils.java

```diff
@@ -0,0 +1,165 @@
+package br.com.italoromeiro.arctouchtest.utils;
+
+import android.Manifest;
+import android.app.AlertDialog;
+import android.app.Dialog;
+import android.content.DialogInterface;
+import android.content.pm.PackageManager;
+import android.os.Bundle;
+import android.support.v4.app.ActivityCompat;
+import android.support.v4.app.DialogFragment;
+import android.support.v7.app.AppCompatActivity;
+import android.widget.Toast;
+
+import br.com.italoromeiro.arctouchtest.R;
+
+
+/**
+ * Utility class for access to runtime permissions.
+ */
+public abstract class PermissionUtils {
```

Comment:
No. I found it inside some example created by the Android guys. I've seen I've forgotten to use the licence. It is already there.